### PR TITLE
glide support

### PR DESCRIPTION
**Pull request by freeformz**
Adds support for [glide](https://github.com/Masterminds/glide) to the buildpack.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 # Heroku Buildpack: Go
 
-This is the [Heroku buildpack][buildpack] for [Go][go].
+This is the offical [Heroku buildpack][buildpack] for [Go][go].
```
Comment (tt) [inline README.md]: "official"

File: bin/compile [hunk 2]
```diff
@@ -22,6 +22,7 @@ NC='\033[0m' # No Color
 CURL="curl -s -L --retry 15 --retry-delay 2" # retry for up to 30 seconds
 
 DefaultGoVersion="go1.6"
+GlideVersion="0.10.2"
```
Comment (lstoll) [inline bin/compile:25]: Is this something we could make overridable by the users environment? Like, do we want to dictate the versions?

Comment (freeformz) [inline bin/compile:25]: I would prefer to constrain these, at least until people have a reason to require different versions.

File: bin/compile [hunk 3]
```diff
@@ -59,18 +60,27 @@ needConcurrency() {
...
+        warn "To install a different package spec set 'heroku.install' in 'vendor/vendor.json'"
+        warn ""
+        warn "For more details see: https://devcenter.heroku.com/articles/go-apps-with-govendor#build-configuration"
+      ;;
+      glide)
+        warn "To install a different package spec for the next build run:"
+        warn ""
+        warn "'heroku config:set GO_INSTALL_PACKAGE_SPEC=\"<pkg spec>\"'"
+        warn ""
+        warn "For more details see: https://devcenter.heroku.com/articles/FIXME"
```
Comment (tt) [inline bin/compile]: I assume this should link to an article.

**Comment by lstoll** [comment]
👍  seems legit to me

File: README.md [hunk 4]
```diff
@@ -95,13 +96,40 @@ spec of `./...` to install.
...
-To change this buildpack, fork it on GitHub. Push changes to your fork, then
-create a test app with `--buildpack YOUR_GITHUB_GIT_URL` and push to it. If you
-already have an existing app you may use `heroku config:add
-BUILDPACK_URL=YOUR_GITHUB_GIT_URL` instead of `--buildpack`.
 
+## Hacking on this Buildpack
+
+To change this buildpack, fork it on GitHub & push changes to your fork. Ensure
+that tests have been added to the `test/run` script and any corresponding fixtures to
+`test/fixutres/<fixutre name>`.
```
Comment (tt) [inline README.md]: "fixtures"/"fixture".

**Comment by tt** [comment]
Looks good to me.

File: bin/compile (hunk 3 above)
Comment (freeformz) [inline bin/compile]: Yeah. Put a placeholder article up, will fill in next.

**Pull request merged**